_candidates_en/SotaroKimura_2.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Sotaro Kimura
+title: "Unified data processing with Spark Structured Streaming"
+length: 40
+audience: Intermediate
+language: Japanese
+twitter: kimutansk
+github: kimutansk
+icon: https://s.gravatar.com/avatar/5d869b93035410ece41ff0d86d8c5c14
+organization: DWANGO Co., Ltd.
+tags:
+  - Big Data / Fast Data
+  - Tools
+suggestions:
+  - People who want to know the difference between batch processing and stream processing
+  - People who want to understand concepts around stream processing 
+  - People who want to know practical uses cases of Spark Structured Streaming
+---
+Stream processing has come far beyond being an alternative to batch processing, and now it is considered as a unified data processing system.
+This session will talk about and review OSS stream processing systems like below,
+and will introduce Spark Structured Processing as a recent example of unified data processing system.
```

Comment:
Consider changing to "+and will introduce Spark Structured Processing as a recent example of an unified data processing system."